### PR TITLE
Include stop orders in gas calculations

### PR DESCRIPTION
**Pull request by witgaw**
- Include stop orders in gas calculations.
- Remove LP shapes.

### CODE REVIEW

**Review by cdummett — DISMISSED** [review]

**Review by barnabee — CHANGES_REQUESTED** [review]
Need to add stop orders to the calculation of gas for a batch. They are a separate "section" of the batch submission.

**Comment by witgaw** [comment]
> Need to add stop orders to the calculation of gas for a batch. They are a separate "section" of the batch submission.

@barnabee is it not enough to just count them along with other cancellations / amendments / submissions? 
https://github.com/vegaprotocol/specs/pull/1806/commits/6f53b4c4a64fa8905e344dc8318ca57ef901b9b6

File: protocol/0079-TGAP-transaction_gas_and_priority.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ gas = min((maxGas/minBlockCapacity)-1,batchGas)
 
 ```go
 gasOliq = network.transaction.defaultgas + peg cost factor x pegs
```
Comment (karlem) [inline protocol/0079-TGAP-transaction_gas_and_priority.md]: The LP submission, amendment and cancelation should not produce any orders anymore so why do we still need this?

Comment (witgaw) [reply protocol/0079-TGAP-transaction_gas_and_priority.md]: Good point, the gas is the same as for the the regular order submission so doesn't really matter, but I can remove that section.

**Review by barnabee — APPROVED** [review]

**Pull request merged**